crates/aptos/src/governance/mod.rs

```diff
@@ -226,17 +229,18 @@ pub struct SubmitVote {
 
     #[clap(flatten)]
     pub(crate) txn_options: TransactionOptions,
-    #[clap(flatten)]
-    pub(crate) pool_address_args: PoolAddressArgs,
+    /// Comma separated list of pool addresses.
+    #[clap(long)]
+    pub(crate) pool_addresses: String,
```

Comment:
use `multiple_values=true` and then you don't have to parse commas, and add `load_account_arg` as the parser from one of the other locations.  Then you'll just get AccountAddresses really easily (space separated)

Reply:
Updated